apps/meteor/app/models/server/models/LivechatDepartmentAgents.js

```diff
@@ -188,7 +184,7 @@ export class LivechatDepartmentAgents extends Base {
 			},
 		};
 
-		const bot = this.findAndModify(query, sort, update);
+		const bot = this.findOneAndUpdate(query, update, { sort, returnNewDocument: 'after' });
```

Comment:
this should use `rawCollection()` or this method/operation should be moved to a raw model